tests/unit/EntityMenu.spec.ts

```diff
@@ -302,35 +302,27 @@ describe('Entity Menu - Amagamate button tests', () => {
       propsData: { businessId: 'BC1234567' }
     })
 
-    expect(wrapper.find('#amalgamate-button').exists()).toBe(true)
-    expect(wrapper.find('#amalgamate-button').text()).toBe('Amalgamate')
-    expect(wrapper.find('#amalgamate-button').classes()).not.toContain('v-btn--disabled') // enabled
-    wrapper.destroy()
-  })
-
-  it('don\'t display Amalgamate button if business is historical', async () => {
```

Comment:
This test is obsolete as the is historical check is now outside (more actions dropdown checks for that).